packages/choice-input/src/ChoiceGroupMixin.js

```diff
@@ -179,5 +167,22 @@ export const ChoiceGroupMixin = dedupeMixin(
           );
         }
       }
+
+      /**
+       * @override FormGroupMixin
+       */
```

Comment:
This was moved from the RadioGroup? to ensure that only a single radio can get checked at once?

Reply:
Before it added its own model-value-changed event listener, now it hooks into this callback provided by FormControlMixin (best way to control timing issues)